VadeCloud/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Could you add an entry in the changelog (like [this one](https://github.com/SEKOIA-IO/automation-library/blob/746698e4608fb627f1922f4825cfe5c3d9c9256a/Duo/CHANGELOG.md?plain=1#L23-L27) in Duo)?